test/parallel/test-http-mutable-headers.js

```diff
@@ -43,8 +43,9 @@ const s = http.createServer(common.mustCall((req, res) => {
   switch (test) {
     case 'headers':
       // Check that header-related functions work before setting any headers
-      // eslint-disable-next-line no-restricted-properties
-      assert.deepEqual(res.getHeaders(), {});
+      const headers = res.getHeaders();
+      const objLike = Object.create(null);
```

Comment:
I agree with @thefourtheye, it's still considered an object [according to the spec](https://www.ecma-international.org/ecma-262/6.0/#sec-ordinary-object-internal-methods-and-internal-slots).